chart/templates/_helpers.tpl

```diff
@@ -204,6 +204,13 @@ Return the absolute path where the GCP Buckets Service Account Key Secret will b
 {{- printf "%s/%s" (include "carto.googleCloudStorageServiceAccountKey.secretMountDir" .) (include "carto.googleCloudStorageServiceAccountKey.secretMountFilename" .) -}}
 {{- end -}}
 
+{{/*
+Create the name of the service account to use for Carto common deployments to connect to google apis
+*/}}
+{{- define "carto.commonGcpApi.serviceAccountName" -}}
+{{- printf "%s-gcp-api" (.Chart.Name) | trunc 63 | trimSuffix "-" -}}
```

Comment:
We should allow to configure it in the same way that it could be configured in the other components.